VocaDbModel/Domain/Discussions/DiscussionTopic.cs

```diff
@@ -60,22 +59,18 @@ public virtual IList<DiscussionComment> Comments
 			}
 		}
 
-		public virtual string Content
-		{
-			get => content;
-			set
-			{
-				ParamIs.NotNullOrEmpty(() => value);
-				content = value;
-			}
-		}
+		public virtual IEnumerable<DiscussionComment> Comments => AllComments.Where(c => c != FirstComment).Where(c => !c.Deleted);

```

Comment:
The `FirstComment` is regarded as the content of a topic, therefore, we exclude it in `Comments`.

Reply:
Please add a comment that mentions this, so we will remember it even merging the PR.